cmd/peer-rpc.go

```diff
@@ -43,23 +44,33 @@ type DeleteBucketArgs struct {
 // DeleteBucket - handles delete bucket RPC call which removes all values of given bucket in global NotificationSys object.
 func (receiver *PeerRPCReceiver) DeleteBucket(args *DeleteBucketArgs, reply *AuthRPCArgs) error {
 	globalNotificationSys.RemoveNotification(args.BucketName)
```

Comment:
Perhaps as another cleanup we can rename `RemoveNotification` to `Remove` as well?

Reply:
may be. not in this PR context.